source/includes/node.md

```diff
@@ -59,6 +60,7 @@ last_metric | Last reported metric from the node (see table `Node Last Metric at
 current_num_containers | The actual number of containers deployed in this node
 last_seen | Date and time of the last time the node was contacted by Tutum
 public_ip | The public IP allocated to the node
+tunnel | The SSL tunnel for accessing the docker daemon
```

Comment:
`If the node does not accept incoming connections to port 2375, the address of the reverse tunnel to access the docker daemon, or `null` otherwise`